code/modules/mining/lavaland/ash_tree.dm

```diff
@@ -0,0 +1,82 @@
+//Houses the ash tree, a lava land tree that has been burning for quite some time making a maple like sweetener.
+
+/obj/structure/flora/ashtree
+	gender = PLURAL //same as other tree
+	icon = 'icons/obj/lavaland/ash_tree.dmi'
+	icon_state = "ashtree"
+	var/sap_icon_state = "ashtree_maple" //Are icon when we are full of honey or other sap
+	var/tabbed_icon_state = "ashtree_maple" //What we look like when tapping
+	name = "ashed tree"
+	desc = "A once large tree now burnt like the lands around it."
+	density = TRUE
+	pixel_x = -16
+	layer = FLY_LAYER
+	var/coal_amount = 5 //amout of coal in are tree, simular to logs
+	var/sap = FALSE //Do we have sap?
+	var/are_sap = /datum/reagent/consumable/honey //What reagent we have
```

Comment:
var/sap_type please. Also, put comments like this

/// What reagent our sap is
var/sap_type = /datum/reagent/consumable/honey

So the code doumentation generator works properly.